peps/pep-0501.rst

```diff
@@ -223,8 +227,13 @@ The key differences between f-strings and t-strings are:
 * unlike f-strings (where conversion specifiers are handled directly in the compiler),
   t-string conversion specifiers are handled at rendering time by the rendering function
 * the new ``!()`` conversion specifier indicates that the field expression is a callable
-  that should be called when using the default :func:`format` rendering function. This specifier
-  is specifically *not* being added to f-strings (since it is pointless there).
+  that should be called when using the default :func:`format` rendering function. This
+  specifier is specifically *not* being added to f-strings (since it is pointless there).
+* a second ``!`` is allowed in t-string conversion specifiers (with any subsequent text
+  being ignored) as a way to allow custom template rendering functions to accept custom
+  conversion specifiers without breaking the default :func:`TemplateLiteral.render`
```

Comment:
We can't link to something that doesn't exist yet :) 

```suggestion
  conversion specifiers without breaking the default ``TemplateLiteral.render``
```

Reply:
```suggestion
  conversion specifiers without breaking the default :func:`!TemplateLiteral.render`
```

Meant to use the exclamation mark to avoid rendering that.